index.html

```diff
@@ -1,70 +1,443 @@
+      <div class="deatails">
+        <div class="shot3"></div>
+        <div class="bo">
+          <header class="pro-title">
+            <h2>Tonic</h2>
+          </header>
+          <div class="item-parrent">
+            <div class="item1">
+              <p class="conopy">CONOPY</p>
+            </div>
+            <div class="item2"></div>
+            <div class="item3">
+              <p class="back">Back End Dev</p>
+            </div>
+            <div class="item4"></div>
+            <div class="item5">
+              <p class="year">2015</p>
+            </div>
+          </div>
+          <div class="text">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="tags">
+            <ul class="ul">
+              <li class="html">
+                <p class="h">HTML</p>
+              </li>
+              <li class="css">
+                <p class="c">CSS</p>
+              </li>
+              <li class="js">
+                <p class="j">JavaScript</p>
+              </li>
+            </ul>
+          </div>
+
+          <div class="action">
+            <div class="boxx">
+              <button class="see-btn">
+                <div class="see">
+                  <p class="se">See Project</p>
+                </div>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="deatails2">
+        <div class="shot4"></div>
+        <div class="bo">
+          <header class="pro-title2">
+            <h2>Multi-Post Stories</h2>
+          </header>
+          <div class="item-parrent">
+            <div class="item1">
+              <p class="conopy">CONOPY</p>
+            </div>
+            <div class="item2"></div>
+            <div class="item3">
+              <p class="back">Back End Dev</p>
+            </div>
+            <div class="item4"></div>
+            <div class="item5">
+              <p class="year">2015</p>
+            </div>
+          </div>
+          <div class="text">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="tags">
+            <ul class="ul">
+              <li class="html">
+                <p class="h">HTML</p>
+              </li>
+              <li class="css">
+                <p class="c">CSS</p>
+              </li>
+              <li class="js">
+                <p class="j">JavaScript</p>
+              </li>
+            </ul>
+          </div>
+
+          <div class="action">
+            <div class="boxx">
+              <button class="see-btn">
+                <div class="see">
+                  <p class="se">See Project</p>
+                </div>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] As the project required, the section of the projects should be styled using CSS Grid. Your section is very good styled but kindly add the Grid layout to this section to meet the project requirements.